packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -7,6 +7,9 @@
 @Tags(<String>['reduced-test-set'])
```

Comment:
I have not been able to give this a full review yet, but are there golden file tests here? (We should see if there are golden file tests in the checkbox PR as well)
Searching for the `matchesGoldenFile` method in the test file is a quick way to check. Having image tests for these types of changes are great for catching regressions.
There is documentation on using these types of tests in flutter/flutter in https://github.com/flutter/flutter/blob/master/docs/contributing/testing/Writing-a-golden-file-test-for-package-flutter.md

Reply:
`CupertinoCheckbox` doesn't have golden tests, but `CupertinoSwitch` does. Whenever I run the golden tests in `CupertinoSwitch`, I get the message:

```
No expectations provided by Skia Gold for test: cupertino.switch.[xxxx].png. This may be a new test. If this is an unexpected result, check https://flutter-gold.skia.org.
```